catalog/.env.example

```diff
@@ -0,0 +1,36 @@
+# Example env file: save as .env (ignored by git) and edit to use.
```

Comment:
my comment looks to have dropped. for this PR i'd like to keep `config.js` as is and backlog introducing a .env.* fils. current system is working :). many of the vars set below are usually set as part of registry dev config.